projects/addon-table/components/table/directives/sortable.directive.ts

```diff
@@ -29,12 +29,6 @@ export class TuiSortableDirective<T extends Partial<Record<keyof T, any>>>
         this.th.sorter = this.sorter;
     }
 
-    ngDoCheck(): void {
-        if (this.match && this.table.sorter !== this.sorter) {
-            this.table.updateSorter(this.sorter);
-        }
-    }
```

Comment:
@waterplea pay attention plz, it created some race (double updateSorter call), but I didn't see its purpose

Reply:
ok
I finally found the problem with doCheck

there are `match` getter in sortable directive, and first time after clicking on `th` there is a wrong value in `this.sortBy.tuiSortBy` https://github.com/taiga-family/taiga-ui/blob/main/projects/addon-table/components/table/directives/sortable.directive.ts#L39
I suggest manually kicking it after clicking on `Th`, then everything seems to be ok